presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -1266,4 +1266,62 @@ private void assertGeomFromBinary(String wkt)
     {
         assertFunction(format("ST_AsText(ST_GeomFromBinary(ST_AsBinary(ST_GeometryFromText('%s'))))", wkt), VARCHAR, wkt);
     }
+
+    @Test
+    public void testGeometryJsonConversion()
+    {
+        // empty geometries
+        // JTS library has trouble creating correct JSON for empty, non-multi geometry,
+        // see https://github.com/locationtech/jts/issues/411
+        // the following tests has been commented out.
+//        assertGeoFromJson("POINT EMPTY");
+//        assertGeoFromJson("LINESTRING EMPTY");
+//        assertGeoFromJson("POLYGON EMPTY");
```

Comment:
This is going to be a problem. I will see if there's any quick fix or workarounds we can do.

Reply:
It looks like the advise is that empty atomic (non-multi) geometries should return `null`: http://lists.geojson.org/pipermail/geojson-geojson.org/2013-October/000839.html

We can implement this by checking for those cases explicitly, and having `GeometryUtils.jsonFromJtsGeometry` return an `Optional<String>`.  In GeoFunctions, if it's empty, return null.  Else return the contained value.